src/nunavut/lang/c/support/nunavut.h

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ * UAVCAN generic serialization support routines.                                                                     +-+ +-+
```

Comment:
```suggestion
 * UAVCAN generic serialization support routines.                                                             +-+ +-+
```

Reply:
I'm not sure what the change is here, I can't see what it is in the github viewer.

Reply:
I believe it was an over-long line. I fixed that issue anyways